.github/workflows/common_hive_reports.yaml

```diff
@@ -59,8 +62,15 @@ jobs:
       - name: Setup Hive
         run: make setup-hive
 
+      - name: Set custom hive flags
+        run: |
+          if [ "${{ matrix.test.simulation }}" == "ethereum/eest/consume-engine" ]; then
+            EEST_FIXTURES=$(cat cmd/ef_tests/blockchain/.fixtures_url)
```

Comment:
Sorry, I know I suggested this, but let's try to avoid custom code inside the Hive repository since we're going to use the official one.

Reply:
This is actually needed to run the tests (its telling the hive test where to download the fixtures from, which is the actual execution-spec-tests repo), the custom code I added (the error mapper) is in the hive fork version